populate/src/main/java/bio/terra/pearl/populate/dto/site/SiteContentPopDto.java

```diff
@@ -2,14 +2,15 @@
 
 import bio.terra.pearl.core.model.site.SiteContent;
 import bio.terra.pearl.populate.dto.FilePopulatable;
-import java.util.HashSet;
-import java.util.Set;
+
+import java.util.ArrayList;
+import java.util.List;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter @Setter @NoArgsConstructor
 public class SiteContentPopDto extends SiteContent implements FilePopulatable {
```

Comment:
outside of the scope of this PR for sure, but I noticed there were a lot of fields (e.g. "sections" from the extended SiteContent) which are serialized but seem to have no purpose now with the DTO. Is there any way to flag them to not serialize? Or is it more work than it's worth?

Reply:
Great question.  Our models, like SiteContent, serve double (or even triple) duty right now.  they serve as 
1. the definition of what's stored in the DB
2. the definition of what's stored in the populate flat files
3. the definition of what's sent from the server to the client. 

In this case, the "sections" aren't used in populate serializing, but they are still very much used for sending from the server to the client. 

We do it this way because it was faster and simpler to get the codebase started that way, and at the beginning, there wasn't much difference between all the differeny uses.  As the codebase matures, we'll likely want to create explicit DTOs for each model for each of those purposes, instead of using bits and pieces for each.